src/Tests/AzureClientTests.cs

```diff
@@ -150,7 +156,7 @@ public void TestManualTargets()
             // set up the mock workspace
             var azureWorkspace = azureClient.ActiveWorkspace as MockAzureWorkspace;
             Assert.IsNotNull(azureWorkspace);
-            azureWorkspace?.AddProviders("ionq", "honeywell", "unrecognized");
+            azureWorkspace?.AddProviders("ionq", "quantinuum", "unrecognized");
```

Comment:
should we keep `honeywell` scenario here to make sure it continues to work?

Reply:
I believe that's covered in other tests, while this one makes sure that IonQ works when Quantinuum neé Honeywell is also added. That said, happy to add both providers here to be sure to cover that case when using IonQ as well.